### PR TITLE
feat: add letter spacing to Text elements

### LINKED ISSUE #3: Support `letterSpacing` in text
We already have a input and icon ready for the `letterSpacing` property, but it's not actually supported yet:

Screenshot 2023-12-27 at 13 04 06

### PR DESCRIPTION
**Pull request by vshakitskiy**
fixed #3

### CODE REVIEW

File: apps/dashboard/src/components/RightPanel/FontSection.tsx [hunk 1]
```diff
@@ -70,7 +70,12 @@ export function FontSection({ selectedElement }: FontSectionProps) {
         }} type="number" value={selectedElement.lineHeight}>
           <LineHeightIcon />
         </Input>
-        <Input max={5} min={0} onChange={() => { updateElement(selectedElement); }} type="number" value={0}>
+        <Input max={5} min={0} onChange={value => {
```
Comment (QuiiBz) [inline apps/dashboard/src/components/RightPanel/FontSection.tsx:73]: Let's also allow going down to `-10` and up to 10, just to allow more customization.

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks again! One small comment:

File: apps/dashboard/src/components/RightPanel/FontSection.tsx (hunk 1 above)
Comment (vshakitskiy) [reply apps/dashboard/src/components/RightPanel/FontSection.tsx:73]: Yea, I'll make it works now.

**Comment by vshakitskiy** [comment]
Now works fine.

**Review by QuiiBz — APPROVED** [review]
LGTM, I just fixed the typecheck error that occurred because of the rebase.

**Pull request merged**